torch/_refs/__init__.py

```diff
@@ -2742,6 +2810,7 @@ def _reshape_view_helper(a: TensorLikeType, *shape, allow_copy: bool) -> TensorL
         # specify the same number of elements above
         accum = a_.shape[idx]
         end = idx
+        return prims.reshape(a, shape)
```

Comment:
@Chillee what's the permanent strategy for this

Reply:
Yeah this works on master.

Reply:
This diff appears to still be necessary, without it, bert fails with

```
  File "/data/users/ezyang/pytorch-tmp/torch/fx/interpreter.py", line 243, in call_function                                       
    return target(*args, **kwargs)                                 
TypeError: must be real number, not torch._C.SymIntNode                                                                               
                                                                   
While executing %sqrt : [#users=1] = call_function[target=math.sqrt](args = (%size_23,), kwargs = {})
Original traceback:                                                
Module stack: {'self_transformer_blocks_0__input_sublayer': 'SublayerConnection', 'self_transformer_blocks_0__lambda_module_attention_
attention': 'Attention'}                                                                                                              
  File "/data/users/ezyang/benchmark/torchbenchmark/models/BERT_pytorch/bert_pytorch/model/attention/single.py", line 16, in forward
    / math.sqrt(query.size(-1))                                                                                                       
 |   File "/data/users/ezyang/benchmark/torchbenchmark/models/BERT_pytorch/bert_pytorch/model/attention/multi_head.py", line 41, in fo
rward

```

Reply:
gonna try to fix this